src/Parallelotope.jl

```diff
@@ -0,0 +1,125 @@
+export Parallelotope
+
+"""
+    Parallelotope{N<:Real, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}}
+     <: AbstractZonotope{N}
+
+Type that represents a parallelotope.
+
+### Fields
+
+- `offset`     -- vector; each element is the offset of the corresponding constraint
+- `directions` -- matrix ``D``; each row is the direction of two corresponding
+                  constraints
+
+### Notes
+``D_{i} = -D_{i+n}`` for ``i = 1, ..., n`` with ``n`` rows in ``D``
+
+"""
+struct Parallelotope{N<:Real, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}} <: AbstractZonotope{N}
+    offset::VN
+    directions::MN
+    center::VN
+
+    # default constructor with dimension check
+    function Parallelotope{N, VN, MN}(o::VN, D::MN, c::VN) where {N<:Real,
+         VN<:AbstractVector{N}, MN<:AbstractMatrix{N}}
+        @assert length(c) == checksquare(D)
+        return new{N, VN, MN}(o, D, c)
+    end
+end
+
+# convenience constructor without type parameter
+Parallelotope(o::VN, D::MN, c::VN) where {N<:Real,
+                                   VN<:AbstractVector{N},
+                                   MN<:AbstractMatrix{N}} = 
+                                   Parallelotope{N, VN, MN}(o, D, c)
+
+"""
+    center(P::Parallelotope{N})::Vector{N} where {N<:Real}
+
+Return the center of a parallelotope.
+
+### Input
+
+- `P` -- parallelotope
+
+### Output
+
+The center of the parallelotope.
+"""
+function center(P::Parallelotope{N})::Vector{N} where {N<:Real}
```

Comment:
```suggestion
function center(P::Parallelotope{N}) where {N<:Real}
```
(same in the documentations and the other getters)